package.json

```
@@ -0,0 +1,86 @@
+{
+    "name": "chowly",
```

Maybe we should pick another name and also use the namespace.
```suggestion
    "name": "@goparrot/chowly-api-client",
```

---

was inspired by google, they have almost the same naming for this kind of features

---

Then to follow the convention of @goparrot packages, we can use `@goparrot/chowly`.